packages/design-system/src/SegmentedControl/SegmentedControl.styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+
+export const StyledSegmentedControl = styled.div<{
+  isDisabled?: boolean;
+}>`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: var(--ads-v2-spaces-2);
+  background-color: var(--ads-v2-color-bg-muted);
+  color: var(--ads-v2-color-fg);
+  padding: var(--ads-v2-spaces-1);
+  border-radius: var(--ads-v2-border-radius);
+`;
+
+export const ControlContainer = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+  border-radius: var(--ads-v2-border-radius);
+  background-color: transparent;
```

Comment:
--ads-vs-color-bg

Reply:
no this needs to be transparent, this is a placeholder later changes to --ads-v2-color-bg